device/bgp/bgplearnedinfofilter.yaml

```diff
@@ -0,0 +1,18 @@
+components:
+  schemas:
+    Bgp.LearnedInformationFilter:
+      x-status: under-review
+      description: >-
+        Configuration for controlling storage of BGP learned information recieved from the DUT.
```

Comment:
shouldn't it be `peer` instead of `DUT` ?

Reply:
Updated to peer from DUT .